From 9e61f0f2699f65fdd55ec04b2134b85391ede634 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Fri, 3 Jun 2022 15:09:00 -0600
Subject: [PATCH] Fix collections import for Python 3.8 compatibility

---
 recordtype.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/recordtype.py b/recordtype.py
index 3fd9d4d..00996a3 100644
--- a/recordtype.py
+++ b/recordtype.py
@@ -32,7 +32,10 @@
 
 import sys as _sys
 from keyword import iskeyword as _iskeyword
-from collections import Mapping as _Mapping
+if _sys.version_info[:2] >= (3, 8):
+    from collections.abc import Mapping as _Mapping
+else: 
+    from collections import Mapping as _Mapping
 
 from six import exec_, string_types
